runtime/j9vm/jvm.c

```diff
@@ -3968,12 +3969,15 @@ JVM_LoadLibrary(const char *libName, jboolean throwOnFailure)
 			UDATA handle = 0;
 			UDATA flags = J9_ARE_ANY_BITS_SET(javaVM->extendedRuntimeFlags, J9_EXTENDED_RUNTIME_LAZY_SYMBOL_RESOLUTION) ? J9PORT_SLOPEN_LAZY : 0;
 			UDATA slOpenResult = j9sl_open_shared_library((char *)libName, &handle, flags);
-
 			Trc_SC_LoadLibrary_OpenShared(libName);
+			attemptedLoad = TRUE;
+
+#if JAVA_SPEC_VERSION < 17
```

Comment:
I think this deserves a comment explaining why we don't want to try again for Java 17+.